os/android/iahwc2.cpp

```diff
@@ -395,10 +401,12 @@ HWC2::Error IAHWC2::HwcDisplay::AcceptDisplayChanges() {
 
 HWC2::Error IAHWC2::HwcDisplay::CreateLayer(hwc2_layer_t *layer) {
   supported(__func__);
+  layer_idx_ = display_.GenerateLayerHash(layer_idx_);
```

Comment:
should it be display_->GenerateLayerHash

Reply:
Yeah, thanks. 👍